docs/team/k-hian.md

```diff
@@ -5,7 +5,7 @@ title: Kay Hian's Project Portfolio Page
 
 ### Project: ResearchRoster
 
-ResearchRoster is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
```

Comment:
i think we should keep the word "address book" here so that it's easier to justify that we're using an address book as a common noun later when they inevitably call us out for reusing the AddressBook class in our uml diagrams in PE lol

Reply:
oh wait this is the kayhian.md document haha.... is there anywhere in our UG that we can write like this?